### PR TITLE
Split RadioButtons tests into smaller chunks and workloads

### LINKED ISSUE #2160: RadioButtons tests take too long
Sometime our test pass times out on RadioButtons tests  when it requires retrying some of the unreliable tests. We could either make the tests more reliable, slit the tests into different work groups, similar to how nav view splits its tests, or increase the timeout alloted to radio button in the GenerateTestProjFile.ps1 or some combination of the 3.

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

I've split up the larger tests into smaller chunks and added them into TestSuite's in order to reduce the time we need to rerun tests and potentially improve test reliability

image

## Motivation and Context

Potentially Fixes #2160 
## How Has This Been Tested?

Ran tests

### CODE REVIEW

**Comment by ranjeshj** [comment]
@chingucoding can you please merge with master, the stability fixes are in now.

**Comment by chingucoding** [comment]
@ranjeshj Good idea, done

File: dev/RadioButtons/InteractionTests/RadioButtonsTests.cs [hunk 1]
```diff
@@ -657,7 +789,8 @@ public void ScrollViewerSettingSelectionDoesNotMoveFocus()
             }
         }
         [TestMethod]
-        public void AccessKeys()
+        [TestProperty("TestSuite", "A")]
+        public void AccessKeysFirstBundle()
```
Comment (StephenLPeters) [inline dev/RadioButtons/InteractionTests/RadioButtonsTests.cs:793]: >public void AccessKeysFirstBundle() [](start = 8, length = 35)

I don't think splitting this test up like this is necessary. It adds a bunch of build up and tear down time that we didn't have before. I think putting this whole test in a suite with less tests is sufficient.

Comment (chingucoding) [reply dev/RadioButtons/InteractionTests/RadioButtonsTests.cs:793]: My idea was that, the longer a test gets, the more likely it is to be unreliable. But if you want I'll revert that splitting up.

On a different note, why can't we rerun single tests and are relying on work items/groups here?😅

**Comment by StephenLPeters** [comment]
I don't think splitting the individual tests is the right approach. I think bucketing the tests into different suites is sufficient...

**Comment by chingucoding** [comment]
@StephenLPeters I've reverted my commit and just added test suite properties to the tests.

File: dev/RadioButtons/InteractionTests/RadioButtonsTests.cs (hunk 1 above)
Comment (StephenLPeters) [reply dev/RadioButtons/InteractionTests/RadioButtonsTests.cs]: We can rerun single tests but the issue we were seeing was that sometimes made the job they were a part of time out.

Comment (chingucoding) [reply dev/RadioButtons/InteractionTests/RadioButtonsTests.cs]: And test suites basically splits that item up, and thus we are getting "more time" here. I see. Thanks for clarification.

**Pull request merged**